app/script/localization/translations/webapp-cs.js

```diff
@@ -491,25 +491,25 @@ z.string.cs.warning_tell_me_how = 'Jak na to';
 z.string.cs.user_availability_available = 'Dostupný';
 z.string.cs.user_availability_none = 'Žádné';
 
-z.string.cs.system_notification_asset_add = 'Sdílel(a) obrázek';
-z.string.cs.system_notification_connection_accepted = 'Přijal(a) váš požadavek na připojení';
-z.string.cs.system_notification_connection_connected = 'Nyní jste připojeni';
-z.string.cs.system_notification_connection_request = 'Žádá o připojení';
-z.string.cs.system_notification_conversation_create = '{{user}} zahájil(a) rozhovor';
-z.string.cs.system_notification_conversation_rename = '{{user}} přejmenoval(a) rozhovor na {{name}}';
-z.string.cs.system_notification_member_join_many = '{{user}} přidal(a) {{number}} kontakty do konverzace';
-z.string.cs.system_notification_member_join_one = '{{user1}} přidal(a) {{user2}} do konverzace';
-z.string.cs.system_notification_member_leave_removed_you = '{{user}} tě odebral(a) z konverzace';
-z.string.cs.system_notification_obfuscated = 'Vám poslal zprávu';
-z.string.cs.system_notification_obfuscated_title = 'Někdo';
-z.string.cs.system_notification_ping = 'Pingnut';
-z.string.cs.system_notification_reaction = '{{reaction}} tvou zprávu';
-z.string.cs.system_notification_shared_audio = 'Sdílel(a) zvukovou zprávu';
-z.string.cs.system_notification_shared_file = 'Sdílel(a) soubor';
-z.string.cs.system_notification_shared_location = 'Sdílel(a) polohu';
-z.string.cs.system_notification_shared_video = 'Sdílel(a) video';
-z.string.cs.system_notification_voice_channel_activate = 'Volá';
-z.string.cs.system_notification_voice_channel_deactivate = 'Volal(a)';
+z.string.cs.notification_asset_add = 'Sdílel(a) obrázek';
+z.string.cs.notification_connection_accepted = 'Přijal(a) váš požadavek na připojení';
+z.string.cs.notification_connection_connected = 'Nyní jste připojeni';
+z.string.cs.notification_connection_request = 'Žádá o připojení';
+z.string.cs.notification_conversation_create = '{{user}} zahájil(a) rozhovor';
+z.string.cs.notification_conversation_rename = '{{user}} přejmenoval(a) rozhovor na {{name}}';
+z.string.cs.notification_member_join_many = '{{user}} přidal(a) {{number}} kontakty do konverzace';
+z.string.cs.notification_member_join_one = '{{user1}} přidal(a) {{user2}} do konverzace';
+z.string.cs.notification_member_leave_removed_you = '{{user}} tě odebral(a) z konverzace';
+z.string.cs.notification_obfuscated = 'Vám poslal zprávu';
+z.string.cs.notification_obfuscated_title = 'Někdo';
+z.string.cs.notification_ping = 'Pingnut';
+z.string.cs.notification_reaction = '{{reaction}} tvou zprávu';
+z.string.cs.notification_shared_audio = 'Sdílel(a) zvukovou zprávu';
+z.string.cs.notification_shared_file = 'Sdílel(a) soubor';
+z.string.cs.notification_shared_location = 'Sdílel(a) polohu';
+z.string.cs.notification_shared_video = 'Sdílel(a) video';
+z.string.cs.notification_voice_channel_activate = 'Volá';
+z.string.cs.notification_voice_channel_deactivate = 'Volal(a)';
```

Comment:
will have quite some conflicts with #2329 (it's more time consuming than I thought)  but will figure it out..

For the next PRs when it comes to new/renames of the strings.. let's make them camelCase right away if needed..